metadata-ingestion/tests/integration/git/test_git_clone.py

```diff
@@ -81,6 +82,15 @@ def test_github_branch():
     assert config.branch_for_clone == "main"
 
 
+def test_sanitize_repo_url():
```

Comment:
Do we have to include this for all doctested entities? Could we scan our `src` directory for all doctests instead? This seems a bit redundant

Reply:
I'm doing this for now, since the doctest scan on `src` fails because of import issues